src/core/crypto/impl/cryptopp/radix.cc

```diff
@@ -26,82 +26,89 @@
  * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,          //
  * STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF    //
  * THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.               //
- *                                                                                            //
- * Parts of the project are originally copyright (c) 2013-2015 The PurpleI2P Project          //
  */
 
-#ifndef SRC_CORE_UTIL_BASE64_H_
-#define SRC_CORE_UTIL_BASE64_H_
+#include "core/crypto/radix.h"
 
-#include <string.h>
+#include <cryptopp/base32.h>
+#include <cryptopp/base64.h>
 
-#include <cstddef>
 #include <cstdint>
+#include <string>
+#include <vector>
+
+namespace kovri
+{
+namespace core
+{
+template <typename T>
+std::string Radix<T>::m_Base32Alphabet("abcdefghijklmnopqrstuvwxyz234567");
+
+std::string Base32::Encode(const std::uint8_t* in, const std::uint64_t len)
+{
+  // Prepare encoder
+  CryptoPP::AlgorithmParameters static const params(CryptoPP::MakeParameters(
```

Comment:
would you be open to adding a documentation line pointing to the CryptoPP Manual page for AlgorithmParameters (https://www.cryptopp.com/docs/ref/class_algorithm_parameters.html)?

Behavior for MakeParameters and AlgorithmParameters was hard to understand from reading the code, until I read the manual page, and it became very obvious. 

Mainly the confusing bit was where AlgorithmParameters overrides the `()` operator to append a key-value pair, and return an AlgorithmParameters ref (allowing one to append an arbitrary number of key-value pairs).

Reply:
>would you be open to adding a documentation line pointing to the CryptoPP Manual page for AlgorithmParameters

This isn't necessary because this type of c++ / crypto++ question is outside the scope of kovri. Adding links to code is generally a bad idea too (links become outdated) and doing so would create a precedent for endless links. Crypto++'s manual merely produces doxygen output so you can read the *same docs* within the code (see `deps/cryptopp`) or visit their website (as you did).